lib/hocon/config_factory.rb

```diff
@@ -12,4 +12,31 @@ def self.parse_file(file_path, options = Hocon::ConfigParseOptions.defaults)
   def self.parse_string(string, options = Hocon::ConfigParseOptions.defaults)
     Hocon::Impl::Parseable.new_string(string, options).parse.to_config
   end
+
+  #
+  # Gets an empty configuration. See also empty_with_description to create an
+  # empty configuration with a description, which may improve user-visible
+  # error messages.
+  #
+  # @return an empty configuration
+  #
+  def self.empty
+    empty_with_description(nil);
+  end
+
+  #
+  # Gets an empty configuration with a description to be used to create a
+  # ConfigOrigin for this Config. The description should
+  # be very short and say what the configuration is, like "default settings"
+  # or "foo settings" or something. (Presumably you will merge some actual
+  # settings into this empty config using Config#withFallback, making
+  # the description more useful.)
+  #
+  # @param originDescription
+  #            description of the config
+  # @return an empty configuration
+  #
+  def self.empty_with_description(origin_description)
```

Comment:
If you change this to `origin_description = nil` you can delete the method immediately above to and rename this method to just `empty`, which is a little cleaner and matches upstream a little closer.